source/winUser.py

```diff
@@ -377,6 +377,11 @@ class GUITHREADINFO(Structure):
 # The height of the virtual screen, in pixels.
 SM_CYVIRTUALSCREEN = 79
 
+# Registers an application wide Window Message so that NVDA can be exited across instances
+WM_EXIT_NVDA = user32.RegisterWindowMessageW("WM_EXIT_NVDA")
+if not WM_EXIT_NVDA:
+	raise WinError()

```

Comment:
Does this result in a log message that is easy to understand?

Reply:
I've added some better logging and comments across all the errors and error handling